tests/analysis/losses/test_losses.py

```diff
@@ -87,7 +87,7 @@ def test_initialize_with_data(self, losses_analysis: type[AnalysisLossesProtocol
 
         _analysis = AnalysisSectionLosses(
             part_of_day=PartOfDayEnum.DAY,
-            resilience_curve_file=test_data.joinpath(
+            resilience_curves_file=test_data.joinpath(
```

Comment:
this is the same file as the one defined in `resilience_curves.conftest.py` (same applies for the one below. Move that fixture one level up so it's accessible from the `tests.analysis.losses` as well.

Reply:
Will do this after merging the other 2 branches to avoid conflicts.

Reply:
Already done in #504